### PR TITLE
Fixes-the-Grammar-Mistake

### LINKED ISSUE #1452: [OTHER]: Grammar Mistake
### What would you like to share?

There is a grammar mistake in Backtracking/permutations page 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AbhineshJha**
### Describe your change:

- Fixes #1452
-Fixes the Grammar Mistake

### CODE REVIEW

File: Backtracking/GeneratePermutations.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Problem Statement: Generate all distinct permutations of a an array (all permutations should be in sorted order);
+ * Problem Statement: Generate all distinct permutations of  an array (all permutations should be in sorted order);
```
Comment (pertrai1) [inline Backtracking/GeneratePermutations.js:2]:
```suggestion
 * Problem Statement: Generate all distinct permutations of an array (all permutations should be in sorted order);
```

Comment (AbhineshJha) [reply Backtracking/GeneratePermutations.js]: done

**Review by raklaptudirm — APPROVED** [review]

**Comment by AbhineshJha** [comment]
please add hacktoberfest-accepted label

**Review by appgurueu — COMMENTED** [review]
Quoting the [Hacktoberfest](https://hacktoberfest.com/participation/) website:

> Avoid submitting low-quality pull/merge requests.

This is a fix for a typo in a comment. It doesn't affect the code or even the meaning of the comment at all. I'm afraid this pretty much makes it a textbook low quality contribution.

We do welcome chores like fixing typos, but please do them in bulk to reduce overhead, and don't expect them to count towards your Hacktoberfest score.

**Comment by raklaptudirm** [comment]
@appgurueu Even though this can't be accepted for hacktoberfest please approve the changes so we can merge.

**Comment by AbhineshJha** [comment]
sorry for inconvenience ! I am new to open source , i am in the learning period , next time this will not happen

**Review by appgurueu — APPROVED** [review]

**Comment by AbhineshJha** [comment]
hey @raklaptudirm merge this !!

**Comment by appgurueu** [comment]
> sorry for inconvenience @appgurueu ! I am new to open source , i am in the learning period , next time this will not happen

That's fine. I suggest you to carefully reread the Hacktoberfest site to know the "rules" as a participant.

I will merge this for now (but I don't think it will count towards Hacktoberfest, given the label matching `\binvalid\b`), but in the future, please don't make PRs to fix single typos; it has way too much overhead. If you do decide to fix typos, fix multiple in one go.

**Pull request merged**